egs2/qasr_tts/local/pretrained.sh

```diff
@@ -0,0 +1,2 @@
+# download pretrained model kan-bayashi/ljspeech_tts_train_transformer_raw_char_tacotron_train.loss.ave
+pretrained="https://zenodo.org/record/4925105/files/tts_train_transformer_raw_char_tacotron_train.loss.ave.zip?download=1"
```

Comment:
Seems redundant, can you merge it to other script?

Reply:
Okay